.github/workflows/release.yml

```diff
@@ -21,3 +21,47 @@ jobs:
       - run: make release-github
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-and-push-image:
```

Comment:
do you think it's worth adding this to a new workflow file that runs on every commit/push? Or is just tagged images sufficient?

Reply:
True, I think only on tagged is sufficient since its the same build process that exists currently, I might cover this if I were to add tests around the containerized gokeyless to the integration tests.